### ISSUE TITLE
GPU-related error: The CUDA driver could not be loaded

### ISSUE THREAD

**Issue by nagdawi** [nightrome/matconvnet-calvin#6]
Hi
I use Matlab 2016a with ubuntu based on your suggestion. Now I work with gcc compiler and the programme run smoothly but when I run dome this error appear.

PS. I don't have GPU
```

>> demo_e2s2_full()
Error using gpuDevice (line 26)
There is a problem with the CUDA driver or with this GPU device. Be sure that you have a supported GPU and
that the latest driver is installed.
Error in CalvinNN/init (line 51)
    gpuDevice(nnOpts.gpus);

Error in CalvinNN (line 30)
            obj.init(nnOpts);

Error in E2S2NN (line 8)
            obj = obj@CalvinNN(net, imdb, nnOpts);

Error in e2s2_wrapper_SiftFlow_full (line 125)
nnClass = E2S2NN(netPath, imdb, nnOpts);

Error in demo_e2s2_full (line 29)
e2s2_wrapper_SiftFlow_full();

Caused by:
    The CUDA driver could not be loaded. The library name used was 'libcuda.so.1'. The error was:
    libcuda.so.1: cannot open shared object file: No such file or directory
```

**Comment by nightrome** [comment]
Great! Now set gpus = [] in
[https://github.com/nightrome/matconvnet-calvin/blob/master/matconvnet-calvin/examples/e2s2/e2s2_wrapper_SiftFlow_full.m#L17](https://github.com/nightrome/matconvnet-calvin/blob/master/matconvnet-calvin/examples/e2s2/e2s2_wrapper_SiftFlow_full.m#L17)

Btw. without GPUs you won't get very far. You probably can't finetune a network in less than a month. But for testing an existing net it should be sort of okay.

**Comment by nagdawi** [comment]
```
>> demo_e2s2_full()
There is a problem with the CUDA driver or with this GPU device. Be sure that you have a supported GPU and
that the latest driver is installed.

Error in parallel.internal.gpu.deviceCount

Error in parallel.gpu.GPUDevice.count (line 26)
            c = parallel.internal.gpu.deviceCount;

Error in gpuDeviceCount (line 14)
    n = parallel.gpu.GPUDevice.count;

Error in SelectIdleGpu (line 16)
numGpus = gpuDeviceCount;

Error in calvinNNDetection (line 45)
nnOpts.gpus = SelectIdleGpu();

Error in demo_frcn (line 23)
calvinNNDetection();

Caused by:
    The CUDA driver could not be loaded. The library name used was 'libcuda.so.1'. The error was:
    libcuda.so.1: cannot open shared object file: No such file or directory

```

**Comment by nagdawi** [comment]
Hi,

Do you mean this will a month to finish?

```
test: epoch 01:   4/2553: 96.4 Hz (104.5 Hz)
test: epoch 01:   5/2553: 99.4 Hz (107.6 Hz)
test: epoch 01:   6/2553: 100.3 Hz (104.0 Hz)
test: epoch 01:   7/2553: 99.8 Hz (96.5 Hz)
test: epoch 01:   8/2553: 98.1 Hz (76.1 Hz)
...
```

Thanks,

**Comment by nightrome** [comment]
Well you are apparently testing the provided network. I was talking about training E2S2 without a GPU.

**Comment by nagdawi** [comment]
@nightrome 
Hi 
I opened a new issue. 

Have a nice day. 🌺
